### ISSUE TITLE
Browse For Existing JDKs

### ISSUE THREAD

**Issue by opticyclic** [microsoft/vscode-java-pack#1260]
## Suggestion

I expect to be able to browse for a JDK and not have to manually type some json.

### What do you want to use this for?
Setting up a  new runtime using a JDK that is extracted locally not with an installer

### What shortcomings exist with current approaches?
I don't like json and I shouldn't have to type out directory name when this use case has been done with a GUI for the last 2 decades.

## Examples

* Download and extract a JDK
* Go to Configure Runtime
* Click a button that brings up a filesystem browser
* Select the directory of the JDK

**Comment by cypher256** [comment]
For your reference, there are Java auto-configuration extensions as below:
https://marketplace.visualstudio.com/items?itemName=Pleiades.java-extension-pack-jdk

**Comment by nickzhums** [comment]
Hi @opticyclic, have you tried using this command: Java: Configure Java Runtime in Command Palette? This will bring up a GUI to configure Java Runtime

**Comment by cypher256** [comment]
@nickzhums `Java: Configure Java Runtime` allows you to drill down and select JAVA_HOME or the JDK installed using the installer, but as @opticyclic says, I don't think you can specify any directory.

The GUI seems to allow you to select the directory detected by [node-jdk-utils](https://github.com/Eskibear/node-jdk-utils/). Also, I think that even the JDK that you edited the json and added to `java.configuration.runtimes` cannot be selected in that GUI.

**Comment by nickzhums** [comment]
Makes sense. We'll think about this, thanks. Issue assigned to me and @jdneo
